qemu/tests/ppc_ic_mode_check.py

```diff
@@ -0,0 +1,48 @@
+import re
+import logging
+
+from virttest import error_context
+from virttest.virt_vm import VMCreateError
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check the interrupt controller mode.
+
+    1) Launch a guest with kernel-irqchip=on/off and ic-mode=xics/xive.
+    2) Get pic info from human monitor and get interrupts info inside guest.
```

Comment:
Should we get pic through QMP monitors?

Reply:
`pic` only available at HMP, there is no alias `query-pic` in QMP.

Reply:
Okay,  I agree with this acquisition method.